Day-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,11 @@
 import React from 'react';
 import './Button.css';
+const ActionButton = () => {
+    return (
+        <div className = 'actionButtons'>
+        <span className = 'clear'>clear</span>
```

Comment:
Instead of having two span, you can pass down the props and re-use the component.